tree/treeplayer/src/TTreeProcessorMT.cxx

```diff
@@ -91,32 +91,141 @@ std::vector<std::vector<Long64_t>> GetFriendEntries(const std::vector<std::pair<
 }
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Get and store the names, aliases and file names of the friends of the tree.
+/// \param[in] tree The main tree whose friends to 
```

Comment:
It should probably be noted somewhere that this routine does not yet handle 'friend of friends' (and of course, also not yet, the fact that a TTree/TChain can be in multiple place in this friend (indirect) collection).

Reply:
will add a note in the doxygen doc